marathon_lb.py

```diff
@@ -1268,6 +1397,9 @@ def get_arg_parser():
                         "number of tasks exposed via marathon-lb.",
                         type=int, default=1000
                         )
+    parser.add_argument("--haproxy-map",
+                        help="Use haproxy maps for domain name to backend"
```

Comment:
I should have been more clear. I was only referring to the help message formatting.